.github/workflows/tests.yml

```diff
@@ -0,0 +1,61 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 12 * * 0'  # run once a week on Sunday
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        # [Python version, tox env]
+        - ["2.7",   "py27"]
+        - ["2.7",   "py27-msgpack1"]
+        - ["2.7",   "py27-zeo4"]
+        - ["3.5",   "py35"]
+        - ["3.5",   "py35-zeo4"]
+        - ["3.6",   "py36"]
+        - ["3.6",   "py36-mtacceptor"]
+        - ["3.6",   "py36-mtacceptor-msgpack1"]
+        - ["3.7",   "py37"]
+        - ["3.7",   "py37-mtacceptor"]
+        - ["3.7",   "py37-mtacceptor-msgpack1"]
+        - ["3.7",   "py37-uvloop"]
+        - ["pypy2", "pypy"]
+        - ["pypy3", "pypy3"]
+        - ["pypy3", "pypy3-mtacceptor"]
+        - ["pypy3", "pypy3-mtacceptor-msgpack1"]
+
+    runs-on: ubuntu-latest
+    name: ${{ matrix.config[1] }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.config[0] }}
+    - name: Pip cache
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ matrix.config[0] }}-${{ hashFiles('setup.*', 'tox.ini') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-${{ matrix.config[0] }}-
+          ${{ runner.os }}-pip-
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox
+    - name: Test
+      run: tox -e ${{ matrix.config[1] }}
+    - name: Coverage
+      if: matrix.config[1] == 'coverage'
```

Comment:
I have not used gha with matrix, but I think this gets never true.

Sorry, just noticed this now as I saw we have an open issue about adding coverage, see #123

Reply:
It is there for the moment when we have coverage testing.